db/migrate/20170815105519_add_order_to_order_items.rb

```diff
@@ -0,0 +1,5 @@
+class AddOrderToOrderItems < ActiveRecord::Migration[5.1]
+  def change
+    add_reference :order_items, :order
```

Comment:
add_reference :order_items, :order, foreign_key: true
http://postgresql.ru.net/manual/tutorial-fk.html

Reply:
Here probably should be add_column, because we not always have an order_id in the order_item record.